tests/checks/braces.fish

```diff
@@ -164,13 +167,13 @@ end
 }
 # CHECK: while
 
-{{echo inner}
-    echo outer}
+{ { echo inner }
+    echo outer
+}
```

Comment:
I think these tests in `braces.fish` were added to test parsing and demonstrate what's possible.
Instead of changing them, can you create copies of the affected paragraphs,
and format only one each.